doc/ubuntu_release_process.md

```diff
@@ -196,19 +209,33 @@ If there were a number of releases that were missed you could do
 
 
 ## Daily packaging recipes and ppa ##
-We have daily packaging recipes that upload to the [daily ppa](https://code.launchpad.net/~cloud-init-dev/+archive/ubuntu/daily).  These build Ubuntu packaging on top of trunk.  This differs from trunk built for the given Ubuntu release because the Ubuntu release may have patches applied.
+We have daily packaging recipes which upload to the [daily ppa](https://code.launchpad.net/~cloud-init-dev/+archive/ubuntu/daily).  These build Ubuntu packaging on top of trunk.  This differs from trunk built for the given Ubuntu release because the Ubuntu release may have patches applied.
 
   * [xenial](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-xenial)
-  * [artful](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-artful)
   * [bionic](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-bionic)
+  * [eoan](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-eoan)
   * [devel](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-devel)
 
 ### When the daily recipe build fails ###
 
-The daily recipe for each release checks out master and then merges the ubuntu/$release
+The daily recipe for each release checks out master and then merges the ubuntu/$release and finally merged ubuntu/daily/$release
 and builds the package from there.  From time to time, the patches in the
 ubuntu/$release branch need to be refreshed/updated as upstream/master changes.
 
```

Comment:
Does this need updating to mention that if we update/refresh packages in ubuntu/$release we also need to refresh ubuntu/daily/$release (is that true?)